packages/web-components/fast-foundation/docs/guide/styling-components.doc.md

```diff
@@ -3,4 +3,91 @@ id: styling-components
+| `--base-height-multiplier`             | `number`             | The number of designUnits used for component height at the base density.   |
+| `--base-horizontal-spacing-multiplier` | `number`             | The number of designUnits used for horizontal spacing at the base density. |
+| `--corner-radius`                      | `number`             | The corner radius of controls.                                             |
+| `--outline-width`                      | `number`             | The width of the outline of outline controls.                              |
+| `--focus-outline-width`                | `number`             | The width the focus indicator.                                             |
+| `--disabled-opacity`                   | `number`             | Opacity of disabled controls.                                              |
+
+## Conditional Stylesheets
+### Forced-colors Stylesheets
+FAST has a commitment to facilitating accessible web experiences and [forced-colors](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors) support is a core tenant of that commitment. `@microsoft/fast-components` exports the `forcedColorsStylesheetBehavior` utility to provide a simple mechanism to apply forced-color stylesheets without bloating the component stylesheet in non-forced-color environments:
+
+__Example: Forced-colors stylesheets__
+```js
+import { forcedColorsStylesheetBehavior } from "@microsoft/fast-foundation";
+import { SystemColors } from "@microsoft/fast-web-utilities";
+const styles = css`
+    /* ... */
+`.withBehaviors(
+    forcedColorsStylesheetBehavior(css`
+        :host {
+            background: ${SystemColors.Canvas};
+            color: ${SystemColors.CanvasText};
+        }
+        /* ... */
+    `)
+)
+```
+
+### MatchMedia Stylesheets
+`forcedColorsStylesheetBehavior` builds is an implementation of the `matchMediaStylesheetBehaviorFactory, which can be used to create custom implementations of stylesheets that will be conditionally attached based on a [MediaQueryList](https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList).
+
+__Example: Using the `matchMediaStylesheetBehaviorFactory`__
+```js
+import { matchMediaStylesheetBehaviorFactory } from "@microsoft/fast-foundation";
+const query = Window.matchMedia('(max-width: 600px)');
+const maxWidthStylesheetBehavior = matchMediaStylesheetBehaviorFactory(query)
+const styles = css`
+    /* ... */
+`.withBehaviors(
+    maxWidthStylesheetBehavior(css`
+        body {
+            color: red;
+        }
+    `)
+)
+```
+
+## Hiding Undefined Elements
+Custom Elements that have not been [upgraded](https://developers.google.com/web/fundamentals/web-components/customelements#upgrades) and don't have styles attached can still be rendered by the browser but they likely do not look how they are supposed to. To avoid a *flash of un-styled content* (FOUC) you should be sure to visually hide Custom Elements if they have not been *defined*:
```

Comment:
remove the words  "you should be sure to " after (FOUC).  So will read instead ... ")FOUC) visually hide Customer Elements if they have not been defined"